frontend/src/components/header.tsx

```diff
@@ -40,12 +52,37 @@ const SearchInputBase = styled(InputBase)({
   zIndex: 2,
 });
 
+interface StyledDrawerProps {
+  drawerOpened : boolean,
+}
 
+const StyledDrawer = styled(MuiDrawer)({
+  flexShrink: 0,
+  whiteSpace: 'nowrap',
+  boxSizing: 'border-box',
+  anchor: 'left',
+}, ({ drawerOpened }: StyledDrawerProps)=> ({
+  ...(drawerOpened && {
+    width: 60,
+    '& .MuiDrawer-paper': { width: 60 },
+  }),
+  ...(!drawerOpened && {
+    width: 120,
+    '& .MuiDrawer-paper': 120,
+  }),

```

Comment:
혹시 아래 1안 2안 중 동작하는 것이 있을까요? 3항연산자를 사용한 것입니다.
가능하다면 1안이 동작하면 좋을 것 같아요

```suggestion
    width: drawerOpened ? 60 : 120,
    '& .MuiDrawer-paper': { width: drawerOpened ? 60 : 120 },
```

```suggestion
  ...(drawerOpened ? {
    width: 60,
    '& .MuiDrawer-paper': { width: 60 },
  } : {
    width: 120,
    '& .MuiDrawer-paper': { width: 120 },
  }),

```